packages/ses/README.md

```diff
@@ -1,7 +1,17 @@
-# Secure ECMAScript (SES)
+# SES
 
-Secure ECMAScript (SES) is an execution environment that provides fine-grained
-sandboxing with Compartments.
+SES is a slight modification to the JavaScript language that enables programs
```

Comment:
I don't like the phrasing "slight modification". I makes it sound like it is not very significant. Rather, I suspect your point is that it is highly compatible, to the point that a tremendous amount of old code, not written to run under SES, nevertheless does compatibly. We should say this explicitly instead.

Reply:
I think SES as a language isn't a good framing. Inventing a new language is trivial (just add or subtract something!). Inventing a solution that makes JS programs secure is really hard, and really valuable. Our framing and marketing should focus on it being a valuable solution for a particular hard problem.

Reply:
from the SEStival yesterday:

- Proposed definitions
    - A variation of [[JavaScript]] that is largely backwards-compatible that gives programs a leg to stand on in defending their own correctness.
        - Not a language, since it's a tool in the language that removes features from the language.
        - How do you explain adding security to a language by removing features?
    - Making JavaScript easily safe for
        - [[multi-tennancy]]
        - mashups
        - plugins
    - "JavaScript without the parts that others can use to hurt you." - [[Erik Marks]]
    - A runtime enforced compute environment that does better than [[Caja]].
        - We are trying to definitely not include a JS parser in the [[trusted computing base (TCB)]]. (We don't want to trust a parser, anyways).

Reply:
another take:

SES is hardened JavaScript.

SES provides a platform to safely compose mutually distrusting JavaScript components. It supports the Principle of Least Authority, allowing components to cooperate without being excessively vulnerable to each other.

The JavaScript ecosystem consists of millions of packages, but without SES, using them makes you vulnerable to attacks or even just bugs that they may contain. SES lets you grant each component exactly the authority that it needs, limiting your vulnerability to just that authority. For example, consider [the event-stream incident][1]. With SES, if you don’t grant `event-stream` access to the network, then it cannot steal your assets.

[1]: https://www.theregister.com/2018/11/26/npm_repo_bitcoin_stealer/